### PR TITLE
WIP BXMSDOC-4118-master: Added a new chapter about coverage report in the test scenario document.

### PR DESCRIPTION
**Pull request by kaldesai**
[Epic](https://issues.jboss.org/browse/BXMSDOC-4007)
[JIRA](https://issues.jboss.org/browse/BXMSDOC-4118)
[Test Scenario RHPAM](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4118-RHPAM-CR/#test-scenarios-coverage-report-con)
[Test Scenario RHDM](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4118-RHDM-CR/#test-scenarios-coverage-report-con)

Please review Chapter 11.

### CODE REVIEW

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-con.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[id='test-scenarios-coverage-report-con']
+= Coverage report for test scenarios
+
+The test scenario designer provides a clear and coherent way of displaying the generated test results using the `Coverage Report` tab from the right side of the test scenario designer.
+
+The `Coverage Report` tab contains the detailed information about available decisions, tested decisions, percentage of tested decisions, how many times each decision has been evaluated, and which decisions are evaluated for the test scenario. You can also view the pie chart with the percentage representation of the ratio between the successful and failed tests.
+
+You can view the coverage reports from the test Scenario designer for both DMN-based scenarios as well as rule-based scenarios.
```
Comment (danielezonca) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-con.adoc]: I saw you note below about current support only for DMN but I think we should at least move it here. Wdyt?

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Review a coverage report for DMN-based test scenario by following the procedure below to validate your DMN models.
+
+.Prerequisites
+* The DMN-based test scenario template has been created for the selected test scenario.For more information about creating DMN-based test scenarios, see xref:test-designer-create-test-template-dmn-based-proc[].
+
+.Procedure
+. Open the DMN-based test scenario in the test scenario designer.
+. Click `Coverage Report` on the right of the test scenario designer to display the overview of the tests and the scenario status.
+
```
Comment (danielezonca) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: I think we should add a note or similar to specify that decision coverage value consider all decisions successfully executed as covered regardless if there is any `EXPECT` condition on it nor the result of such `EXPECT` conditions (they can all fail from coverage perspective)

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-con.adoc (hunk 1 above)
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-con.adoc]: Ack.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc]: @danielezonca, Can you please explain what is actually needed in that note?

**Pull request closed without merging**